cluster/log-dump/log-dump.sh

```diff
@@ -282,7 +282,7 @@ function dump_nodes_with_logexporter() {
   sed -i'' -e "s@{{.EnableHollowNodeLogs}}@${enable_hollow_node_logs}@g" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
 
   # Create the logexporter namespace, service-account secret and the logexporter daemonset within that namespace.
-  KUBECTL="${KUBECTL:-${KUBE_ROOT}/cluster/kubectl.sh}"
+  KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
```

Comment:
Sometimes KUBECTL is defined as `./cluster/kubectl.sh --match-server-version`. We use that flag for some tests and we wouldn't be able to run logexporter there due to version skew (even though we're able to create daemonset). Wdyt?